frontend/src/containers/CurrentWeather.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react"
+import Box from "../components/Box"
+import { fetchDataProduct } from "../utils"
+import WeatherData from "../components/WeatherData"
+import DataProductLink from "../components/DataProductLink"
+
+const DEFINITION = "draft/Weather/Current/Metric"
+const LOCATIONS = {
+  Helsinki: [60.1699, 24.9384],
+  Tokyo: [35.6762, 139.6503],
+  Lisbon: [38.7223, 9.1393],
+  "New York": [40.7128, 74.006],
+}
+const DEFAULT_CITY = "Helsinki"
+
+export default function CurrentWeather() {
+  const [location, setLocation] = useState(LOCATIONS[DEFAULT_CITY])
+  const [lat, setLat] = useState(LOCATIONS[DEFAULT_CITY][0])
+  const [lon, setLon] = useState(LOCATIONS[DEFAULT_CITY][1])
+
+  const [isFetching, setIsFetching] = useState(false)
+  const [weather, setWeather] = useState({})
+
+  function updateCoords(event) {
+    const city = event.target.value
+    setLocation(city)
+    setLat(LOCATIONS[city][0])
+    setLon(LOCATIONS[city][1])
+  }
+
+  async function fetchWeather(event) {
+    event.preventDefault()
+    setIsFetching(true)
+    const resp = await fetchDataProduct(DEFINITION, "digitalliving", {
```

Comment:
We should swap the source to `ioxio`. If it's not configured, then we need to configure it.

```suggestion
    const resp = await fetchDataProduct(DEFINITION, "ioxio", {
```